modules/tracking/tutorials/tutorial_multitracker.markdown

```diff
@@ -0,0 +1,49 @@
+Using MultiTracker {#tutorial_multitracker}
+==================
+
+Goal
+----
+
+In this tutorial you will learn how to
+
+-   Create a MultiTracker object.
+-   Track several objects at once using the MultiTracker object.
+
+Source Code
+-----------
+
+@includelineno tracking/samples/tutorial_multitracker.cpp
+
+Explanation
+-----------
+
+-#  **Create the MultiTracker object**
+
+    @snippet tracking/samples/tutorial_multitracker.cpp create
+
+    You can create the MultiTracker object and use same tracking algorithm for all tracked object as shown in the snippet.
+    If you should define the tracking algorithm whenever a new object is added to the MultiTracker object.
```

Comment:
I do not really understand what you want to say in this sentence. I think the leading if should be removed, otherwise the whenever is funny.